data/strings/strings.txt

```diff
@@ -5687,6 +5687,53 @@
     zh-Hans = 测试语音指令（TTS，Text-To-Speech）
     zh-Hant = 測試語音指令（TTS、文本轉語音
 
+  [pref_tts_open_system_settings]
+    comment = button to open the system's TTS setting menu
+    tags = android
+    en = Open system TTS settings
```

Comment:
Wrong translations were generated for some languages because of missing "the" article. What about a different text?

```
    en = Speech synthesis system settings
    af = Stelsel spraak sintese instellings
    ar = إعدادات تركيب الكلام النظام
    az = Sistem nitq sintezi parametrləri
    be = Сістэмныя наладкі сінтэзу маўлення
    bg = Настройки на системата за синтез на реч
    ca = Configuració de la síntesi de veu del sistema
    cs = Nastavení systému syntézy řeči
    da = Indstillinger for talesyntese-system
    de = Einstellungen des Sprachsynthesesystems
    el = Ρυθμίσεις συστήματος σύνθεσης ομιλίας
    es = Ajustes del sistema de síntesis de voz
    es-MX = Configuración de síntesis de voz del sistema
    et = Kõnesünteesi süsteemi seaded
    eu = Sistemaren ahots-sintesiaren ezarpenak
    fa = تنظیمات سنتز گفتار سیستم
    fi = Puhesynteesijärjestelmän asetukset
    fr = Paramètres du système de synthèse vocale
    he = הגדרות סינתזת דיבור במערכת
    hi = सिस्टम वाक् संश्लेषण सेटिंग्स
    hu = Beszédszintetizáló rendszer beállításai
    id = Pengaturan sistem sintesis ucapan
    it = Impostazioni del sistema di sintesi vocale
    ja = 音声合成システムの設定
    ko = 음성 합성 시스템 설정
    lt = Kalbos sintezės sistemos nustatymai
    mr = सिस्टम स्पीच सिंथेसिस सेटिंग्ज
    nb = Systeminnstillinger for talesyntese
    nl = Instellingen spraaksynthesesysteem
    pl = Ustawienia systemu syntezy mowy
    pt = Definições do sistema de síntese de voz
    pt-BR = Configurações do sistema de síntese de fala
    ro = Setări ale sistemului de sinteză vocală
    ru = Системные настройки синтеза речи
    sk = Nastavenia systému syntézy reči
    sv = Inställningar för talsyntes
    sw = Mipangilio ya usanisi wa hotuba ya mfumo
    th = การตั้งค่าการสังเคราะห์เสียงพูดของระบบ
    tr = Konuşma sentezleme sistemi ayarları
    uk = Системні налаштування синтезу мови
    vi = Cài đặt tổng hợp giọng nói của hệ thống
    zh-Hans = 语音合成系统设置
    zh-Hant = 系統語音合成設定
```